docs/installation/extending-odc.rst

```diff
@@ -22,9 +22,6 @@ other environment issues.
 .. _datacube/drivers/driver_cache.py: https://github.com/opendatacube/datacube-core/blob/60187e38669d529c55d05a962bd7c5288d906f1b/datacube/drivers/driver_cache.py
 .. _setuptools dynamic service and plugin discovery mechanism: https://packaging.python.org/guides/creating-and-discovering-plugins/#using-package-metadata
 
-.. Broken link
-   .. _setuptools dynamic service and plugin discovery mechanism: http://setuptools.readthedocs.io/en/latest/setuptools.html#dynamic-discovery-of-services-and-plugins
-
 
```

Comment:
The working link for the new API is: https://setuptools.pypa.io/en/latest/pkg_resources.html

Reply:
I'm guessing Sphinx only permits one "anchor" definition per name. Do you want me to replace the link in line 23 with the link you pasted?

Reply:
Actually the one above is probably a better choice.  Nevermind, let's merge as is.